### PR TITLE
BaSP-TG-117: Hook forms in Tasks

### PR DESCRIPTION
**Pull request by JuliMonte5**
Implemented React Hook Forms and Joi libraries in Tasks Formulary.

### CODE REVIEW

File: src/Components/Tasks/Form/index.js [hunk 1]
```diff
@@ -45,9 +51,9 @@ const Form = () => {
   useEffect(() => {
     if (taskId) {
       if (dataTask !== undefined) {
-        setNameValue({ description: dataTask.description });
+        reset({ description: dataTask.description });
```
Comment (Ignacio-Sormani) [inline src/Components/Tasks/Form/index.js:54]: In both reset you are passing the same parameters. Just make an OR in the first if

Comment (JuliMonte5) [reply src/Components/Tasks/Form/index.js:52]: Done

File: src/Components/Tasks/Form/validations.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import joi from 'joi';
```
Comment (GonzaloAsencio) [inline src/Components/Tasks/Form/validations.js:1]: custom the message of this validation

Comment (JuliMonte5) [reply src/Components/Tasks/Form/validations.js:1]: Done

**Review by GonzaloAsencio — APPROVED** [review]

**Review by MaicolFreire001 — APPROVED** [review]
SGTM🚀

**Review by sofihoj — APPROVED** [review]

**Pull request merged**